src/lib/y2storage/planned/assigned_space.rb

```diff
@@ -67,6 +67,13 @@ def partition_type
         end
       end
 
+      # Sum of the weights of all the planned partitions assigned to this space
+      #
+      # @return [Integer]
+      def total_weight
+        partitions.map { |p| p.weight || 0 }.reduce(:+)
```

Comment:
does it make sense to have default in that weight instead? so nil is 0 defined in one place. I cannot see semantics in weight definition, so I am not so sure.

Reply:
Well, this is a sum and 0 is the [identity element](https://en.wikipedia.org/wiki/Identity_element) for addition operations. Since there is nothing like the "canonical default weight" (in fact, the weights only make full sense in concrete contexts, when compared to others), using the identity element of the operation is the most reasonable approach.